### PR TITLE
 Update Collection Specs with CMS Data

### PR DESCRIPTION
**Pull request by nikunjsdev**
https://github.com/adrienshen/saleor-storefront/projects/1#card-39234049

### CODE REVIEW

File: src/views/CollectionIndex/ViewDetails/Page.tsx [hunk 1]
```diff
@@ -4,25 +4,17 @@ import { Link } from "react-router-dom";
...
+  const specsDetails = JSON.parse(props.data.collection.extraFields) || [];
+  const details =
+    specsDetails &&
+    specsDetails.specifications &&
+    specsDetails.specifications.length > 0 &&
+    specsDetails.specifications.map(item =>
+      Object.keys(item).map(key => {
+        return { key, value: item[key] };
+      })
+    );
```
Comment (zshanhui) [inline src/views/CollectionIndex/ViewDetails/Page.tsx]: You can extract this to separate functions and return what you need: `list of key values`.

File: src/views/CollectionIndex/ViewDetails/Page.tsx [hunk 2]
```diff
@@ -38,7 +30,11 @@ export const Page = props => {
       <div>
         <div className="wrapper-header">Set Details</div>
         {details.map((detail, i) => {
```
Comment (zshanhui) [inline src/views/CollectionIndex/ViewDetails/Page.tsx]: Use `idx` instead of `i`

**Review by zshanhui — CHANGES_REQUESTED** [review]

**Comment by zshanhui** [comment]
@nikunjsdev 
Just a few changes to clean things up and then it's LGTM!

File: src/views/CollectionIndex/ViewDetails/Page.tsx (hunk 1 above)
Comment (nikunjsdev) [reply src/views/CollectionIndex/ViewDetails/Page.tsx]: Done

File: src/views/CollectionIndex/ViewDetails/Page.tsx (hunk 2 above)
Comment (nikunjsdev) [reply src/views/CollectionIndex/ViewDetails/Page.tsx]: Done

**Review by zshanhui — APPROVED** [review]

**Pull request merged**